### DISCUSSION TITLE
models/287/DPA-2.3.1-v3.0.0rc0

### DISCUSSION THREAD

**Discussion by giscus** [deepmodeling/AIS-Square#238] [Announcements]
# models/287/DPA-2.3.1-v3.0.0rc0

https://www.aissquare.com/models/detail?pageType=models&name=DPA-2.3.1-v3.0.0rc0&id=287

**Comment by gao1234848** [comment]
请问，冻结出来的模型进行微调再冻结转化成LAMMPS可行吗
